scripts/install_backend_python_libs_test.py

```diff
@@ -602,3 +622,39 @@ def test_pip_install_with_import_error_and_windows_os(self):
         self.assertTrue(
             'https://github.com/oppia/oppia/wiki/Installing-Oppia-%28'
             'Windows%29' in self.print_arr)
+
+    def test_uniqueness_of_normalized_lib_names_in_requirements_file(self):
+        normalized_library_names = set()
+        with python_utils.open_file(common.REQUIREMENTS_FILE_PATH, 'r') as f:
+            lines = f.readlines()
+            for line in lines:
+                trimmed_line = line.strip()
+                if trimmed_line.startswith('#') or len(trimmed_line) == 0:
+                    continue
+                library_name_and_version_string = trimmed_line.split(
+                    ' ')[0].split('==')
```

Comment:
I suggest moving this common code between the tests to a test helper to keep the tests more focused on the behavior that they're verifying.